test/promises_aplus_adapter.js

```diff
@@ -13,11 +14,9 @@ require( "jsdom" ).env( "", function ( errors, window ) {
 		var deferred = jQuery.Deferred();
 
 		return {
-			get promise() {
-				return deferred.promise();
-			},
+			promise: deferred.promise(),
```

Comment:
> tests pass (this is tested on Travis so it wouldn't be green if it failed in any tested Node/io.js)

It wouldn't manifest as failed tests, but as functionality that is now no longer being tested. Maybe I'm misremembering, though... this does look fine, so :+1:.

> I mainly changed it here since I was touching this file as well and having a regular property here means we might be able to run at least a subset of tests in IE8 (not relevant for this branch but still).

Understood and agreed.